locksmith/src/operations/eventOperations.ts

```diff
@@ -52,8 +55,7 @@ export const getEventDetail = async (
     )
   }
 
-  // This is an event, collect event information
-  if (attributes) {
+  if (types.isEvent) {
```

Comment:
We already have this to prevent but is still needed here in the case we move this function 
```
// get event details only when lock is event
  if (isEvent) {
    eventDetail = await getEventDetail(lockAddress, network)
  }
```